lib/state/settings/reducer.js

```diff
@@ -57,12 +67,28 @@ const markdownEnabled = ( state = false, action ) => {
 	return action.markdownEnabled;
 };
 
+const visiblePanes = ( state = [
+	'navigation-bar',
+	'note-list',
+	'note-editor',
+], action ) => {
+	if ( SHOW_PANE !== action.type && HIDE_PANE !== action.type ) {
+		return state;
+	}
```

Comment:
I'm afraid this is needed because there's a `Symbol` action type (`AUTH_SET`?) that breaks the `get()` with `Uncaught TypeError: Cannot convert a Symbol value to a string`.